src/meta/inode.rs

```diff
@@ -0,0 +1,66 @@
+use crate::color::{ColoredString, Colors, Elem};
+use std::fs::Metadata;
+
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+pub struct INode {
+    index: u64,
+    valide: bool,
+}
+
+impl<'a> From<&'a Metadata> for INode {
+    #[cfg(unix)]
+    fn from(meta: &Metadata) -> Self {
+        use std::os::unix::fs::MetadataExt;
+
+        let index = meta.ino();
+
+        Self {
+            index,
+            valide: true,
+        }
+    }
+
+    #[cfg(windows)]
+    fn from(_: &Metadata) -> Self {
+        Self {
+            index: 0,
+            valide: false,
+        }
+    }
+}
+
+impl INode {
+    pub fn render(&self, colors: &Colors) -> ColoredString {
+        if !self.valide {
+            return colors.colorize(String::from("-"), &Elem::SymLink);
+        }
+
+        colors.colorize(self.index.to_string(), &Elem::SymLink)
```

Comment:
It won't be totally bad idea to create a new color maybe? You could just copy the color from `SymLink` or we could probably check with @Peltoche and assign something colorful.

Reply:
added a new color for `inode`, will continue to discuss in the previous comment